src/components/accountDisplay.tsx

```diff
@@ -14,25 +15,33 @@ export class AccountDisplay extends React.Component<AccountDisplayProps, {}> {
     super(props);
   }
 
+  public handleBackClick = () => {
+    this.props.onBackClick(null);
```

Comment:
again, this is somewhat weird.   I don't think the back button should be responsible for setting a value.